app/components/e-panel.vue

```diff
@@ -0,0 +1,543 @@
+       *
+       * Valid values: `000`, `100`, `200`, `300`, `400`
+       */
+      headlineSpacingBottom: {
+        type: String,
+        default: '100',
+        validator(value) {
+          return ['000', '100', '200', '300', '400'].includes(value);
+        },
+      },
+
+      /**
+       * Defines the left spacing of the headline
+       *
+       * Valid values: `000`, `100`, `200`, `300`, `400`
+       */
+      headlineSpacingLeft: {
+        type: String,
+        default: '100',
+        validator(value) {
+          return ['000', '100', '200', '300', '400'].includes(value);
+        },
+      },
+
+      /**
+       * Defines the headline of the panel
+       *
+       */
+      headline: {
+        type: String,
+        default: null,
+      },
+
+      /**
+       * Defines the contour of the panel
+       *
+       * Valid values: `thin`, `thick`
+       */
+      contour: {
+        type: String,
+        default: 'thin',
+        validator(value) {
+          return [
+            'thin',
+            'thick'
+          ].includes(value);
+        },
+      },
+
+      /**
+       * Defines the contour color of the panel
+       *
+       * Valid values: `yellow`, `blue`
+       */
+      color: {
+        type: String,
+        default: 'yellow',
+        validator(value) {
+          return [
+            'yellow',
+            'blue',
+            'gray'
+          ].includes(value);
+        },
+      },
+
+      /**
+       * Defines if the triangle shaped button shall be displayed or not
+       */
+      triangleButton: {
+        type: Boolean,
+        default: false
+      },
+
+      /**
+       * Defines the size of the triangle button
+       *
+       * Valid values: `small`, `big`
+       */
+      triangleButtonSize: {
+        type: String,
+        default: 'small',
+        validator(value) {
+          return [
+            'small',
+            'big'
+          ].includes(value);
+        },
+      },
+
+      /**
+       * Defines the plus color of the plus for the triangle button
+       *
+       * Valid values: `white`, `blue`
+       */
+      triangleButtonPlusColor: {
+        type: String,
+        default: 'white',
+        validator(value) {
+          return [
+            'white',
+            'blue'
+          ].includes(value);
+        },
+      },
+
+    },
+    // data() {
+    //   return {};
+    // },
+
+    computed: {
+      hasHeadline() {
+        return this.headline !== null && this.headline !== '';
+      }
+    },
+    // watch: {},
+
+    // beforeCreate() {},
+    // created() {},
+    // beforeMount() {},
+    // mounted() { },
+    // beforeUpdate() {},
+    // updated() {},
+    // activated() {},
+    // deactivated() {},
+    // beforeDestroy() {},
+    // destroyed() {},
+
+    methods: {
+      mouseOver(event) {
+        const plus = event.target.classList.contains('e-panel__plus') ? event.target : event.target.firstChild;
```

Comment:
- [ ] I would recommend to save the elements inside `data()` on `mounted` and add a new method (e.g. `toggleTriangleHover(hover)` where you add/remove the classes from the cached items.

Your version also works, but I think its a bit flimsy since the event can raise from different elements.